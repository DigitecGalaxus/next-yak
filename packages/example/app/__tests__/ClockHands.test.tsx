/// <reference types="jest" />
import React from "react";
import { render } from "@testing-library/react";
import { ClockHands } from "../ClockHands";

test("renders ClockHands component", () => {
  const { asFragment } = render(<ClockHands />);
  expect(asFragment()).toMatchSnapshot();
});
