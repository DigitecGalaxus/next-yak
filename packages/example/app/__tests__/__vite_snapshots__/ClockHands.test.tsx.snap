// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders ClockHands component 1`] = `
<DocumentFragment>
  <div />
  <div
    class="
  width: 4px;
  height: 40%;
  background: black;
  translate: 0 0 40px;
"
  />
  <div
    class="
  width: 6px;
  height: 30%;
  background: black;
  translate: 0 0 40px;
"
  />
</DocumentFragment>
`;
